scanpy_scripts/lib/_combat.py

```diff
@@ -0,0 +1,51 @@
+"""
+scanpy combat
+"""
+
+import scanpy as sc
+
+# Wrapper for mnn allowing use of non-standard slot
+
+def combat(adata, key=None, key_added=None, layer=None, **kwargs):
```

Comment:
I'm guessing that this is using Combat_seq right? or is it applicable on differential expression results? My understanding is that the origina Combat was written for microarrays continuous real data and is not directly applicable to non-negative discrete count data.

Reply:
This is just a wrapper around Scanpy's own function: https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html, derived from https://github.com/brentp/combat.py.

People got very excited about the discrete thing with sequencing at the start, but my understanding is that subsequent testing indicted that in practice the issue was not such a big deal.